content/blog/haven-2-2-0/index.Rmd

```diff
@@ -0,0 +1,53 @@
+---
+output: hugodown::hugo_document
+
+slug: haven-2-2-0
+title: haven 2.2.0
+date: 2020-05-27
+author: Hadley Wickham
+description: >
+    haven now uses vctrs which means labelled classes will be preserved in
+    tidyr and dplyr operation.
+
+photo:
+  url: https://unsplash.com/photos/45GmTCcW8Hk
+  author: Sergey Nikolaev
+
+categories: [package] 
+tags: [haven]
```

Comment:
Just an FYI, I had been tagging by organisation, too (e.g., tidyverse, r-lib), but I'm totally fine dropping that.